src/core_plugins/kibana/public/dashboard/dashboard_constants.js

```diff
@@ -2,7 +2,10 @@
 export const DashboardConstants = {
   ADD_VISUALIZATION_TO_DASHBOARD_MODE_PARAM: 'addToDashboard',
   NEW_VISUALIZATION_ID_PARAM: 'addVisualization',
-  LANDING_PAGE_PATH: '/dashboard',
-  CREATE_NEW_DASHBOARD_URL: '/dashboard/create',
-  EXISTING_DASHBOARD_URL: '/dashboard/{{id}}'
+  LANDING_PAGE_PATH: '/dashboard/list',
+  // NOTE, the following two urls have to have LANDING_PAGE_PATH as their sub url for
+  // the chrome nav links to work correctly (eg. so navigating away from an opened dashboard, and back to
+  // the dashboard app, will reopen the dashboard, not show the landing page.
```

Comment:
If we're going to change the urls I would vote for:

 - `/dashboards` - plural, shows index
 - `/dashboards/new` - create new dashboard
 - `/dashboard/{id}` - view a specific dashboard
 - `/dashboard/{id}/edit` - edit a specific dashboard